testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+class TextInputAction @Inject constructor(
+  val context: Context
+) {
+  fun hasErrorText(@StringRes expectedErrorTextId: Int): Matcher<TextInputLayout> {
```

Comment:
Add kdoc

Reply:
Rather than passing the `StringRes id`, let's pass String, in this way we don't need to pass context to the constructor.

Reply:
Done